### ISSUE TITLE
Integrate with Fediseer tags

### ISSUE THREAD

**Issue by db0** [LemmyNet/joinlemmy-site#295]
[Fediseer](https://fediseer.com) is a service which allows instance admins to crowdsource and collaborate on instance management. One of the features it has is self-tagging of instances as [you can see here](https://gui.fediseer.com/instances/detail/lemmy.dbzer0.com).

The fediseer also provides a [full REST API](https://fediseer.com/api) anyone can use to retrieve this data. 

I want to suggest that instead of hardcoding instances to the the various topics [as seen here](https://github.com/LemmyNet/joinlemmy-site/blob/0c2ac063a1af150dacc1f90dc1b0377a299a5847/src/shared/components/instances-definitions.ts#L230), you use the tagging functionality to retrieve and discover instances.

If you want to have specific topics only, you could then incentivize the admins to tag their instances with the names you're looking for. For example to retrieve all instances self-tagged as `technology`: https://fediseer.com/api/v1/whitelist?endorsements=0&guarantors=0&tags_csv=technology&page=1&limit=100

Alternatively this would allow you to set up a cascading selection to allow people to discover smaller instances. For example `AI` might be under technology. So someone clicking on technology, will see all instances tagged as `technology` but then see also potential filtering for `AI`, `FOSS`, `Renewable Energy` etc.

Likewise, an instance could natively appear under more tags. For example lemmy.dbzer0.com might appear under Politics > Anarchism, Technology > AI, and Neurodivergence

This will help discoverability of smaller instances without forcing the owners of this repo to constantly keep it updated with new instances manually.

**Comment by dessalines** [comment]
I don't know that we'd want to rely on an external service for the instance list. I'd be happy to link it from the top of that page if you'd like. Feel free to make a PR for that.

**Comment by Nutomic** [comment]
I think this makes sense because the instance list from fediseer is much more actively maintained than the one in joinlemmy. It would also reduce our workload as we wont have to manage the list anymore, or think about including/excluding specific instances. Additionally we could easily list only trustworthy instances using a minimum number of endorsements received, or requiring endorsements from a hardcoded list of trusted instances.

Anyway someone would have to be willing to implement this.

**Comment by dessalines** [comment]
That looks too overkill for our purposes (especially hierarchical tagging). Also we might not want to host everything that's on fediseer.

**Comment by db0** [comment]
You don't have to. You can choose to display only guaranteed instances and with potentially more than X endorsements. You can disallow instances that specific other instances have censured, etc. The idea is to crowdsource the maintenance.

**Comment by db0** [comment]
Boosting this because I feel this is still necessary. The join-lemmy site is so unsuited for the purpose so that people who actively go on social media to promote lemmy **[are actively avoiding it](https://feddit.org/post/7090265)**!

**Comment by dessalines** [comment]
We sort by random and encourage signups to smaller instances on join-lemmy.org to spread out the load, and make the fediverse more sustainable. The ppl in the thread you linked are trying to do the opposite: centralize everything onto suggested large servers.

A lot of those comments seem to be complaining about the US-reddit-slop that dominates /All from a lot of the bigger instances. I don't think there's a way around that, especially if people sign up on instances that use `All` as their default view.

**Comment by db0** [comment]
> The ppl in the thread you linked are trying to do the opposite: centralize everything onto suggested large servers.

Yes, because the current join lemmy is not sufficient and pointing people to some biggish servers is a simpler option for newbies. Fact of the matter is that we have a situation where the people actually doing the outreach, are avoiding your outreach site and doing custom workarounds.
